tools/gitserver/README.md

```diff
@@ -0,0 +1,6 @@
+# Git Server
+
+This is a tool that exposes an HTTP server with git repository, that can be used in e2e tests, to avoid connections with external git providers.
```

Comment:
```suggestion
Git Server is a tool that exposes an HTTP server with a Git repository. It can be used in e2e tests to avoid connections with external Git providers.
```